deployment/CP02PMUO_Deploy.csv

```diff
@@ -1,121 +1,121 @@
 AR-34,,AR-39,,CP02PMUO-SBS01-00-STCENG000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,,CGCON-BSTC01-00009,39.9417,-70.7804,,0,449,
-AR-34,,AR-39,,CP02PMUO-SBS01-01-MOPAK0000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,,CGINS-MOPAK0-99999,39.9417,-70.7804,,0,449,sn not in config
-AR-34,,AR-39,,CP02PMUO-WFP01-00-WFPENG000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGCON-CWFPCP-10304,39.9417,-70.7804,,428,449,
-AR-34,,AR-39,,CP02PMUO-WFP01-01-VEL3DK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-VEL3DK-00033,39.9417,-70.7804,,428,449,
-AR-34,,AR-39,,CP02PMUO-WFP01-02-DOFSTK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-DOFSTK-02726,39.9417,-70.7804,,428,449,
-AR-34,,AR-39,,CP02PMUO-WFP01-03-CTDPFK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-CTDPFK-00127,39.9417,-70.7804,,428,449,
-AR-34,,AR-39,,CP02PMUO-WFP01-04-FLORTK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-FLORTK-01100,39.9417,-70.7804,,428,449,
-AR-34,,AR-39,,CP02PMUO-WFP01-05-PARADK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-PARADK-20582,39.9417,-70.7804,,428,449,
+AR-34,,AR-39,,CP02PMUO-SBS01-01-MOPAK0000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,,CGINS-MOPAK0-99999,39.9417,-70.7804,,0,449,
+AR-34,,AR-39,,CP02PMUO-WFP01-00-WFPENG000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGCON-CWFPCP-10304,39.9417,-70.7804,,425,449,
+AR-34,,AR-39,,CP02PMUO-WFP01-01-VEL3DK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-VEL3DK-00033,39.9417,-70.7804,,425,449,
+AR-34,,AR-39,,CP02PMUO-WFP01-02-DOFSTK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-DOFSTK-02726,39.9417,-70.7804,,425,449,
+AR-34,,AR-39,,CP02PMUO-WFP01-03-CTDPFK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-CTDPFK-00127,39.9417,-70.7804,,425,449,
+AR-34,,AR-39,,CP02PMUO-WFP01-04-FLORTK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-FLORTK-01100,39.9417,-70.7804,,425,449,
+AR-34,,AR-39,,CP02PMUO-WFP01-05-PARADK000,13,1,2019-04-23T14:11:00,2019-10-14T12:01:00,CGMCP-02PMUO-00013,CGVEH-CWFPCP-10304,CGINS-PARADK-20582,39.9417,-70.7804,,425,449,
 AR-39,,,,CP02PMUO-RII01-02-ADCPSL010,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,,CGINS-ADCPSL-22846,39.941912,-70.779573,,426,449,
 AR-39,,,,CP02PMUO-SBS01-00-STCENG000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,,CGCON-BSTC01-00005,39.941912,-70.779573,,0,449,
 AR-39,,,,CP02PMUO-SBS01-01-MOPAK0000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,,CGINS-MOPAK0-11358,39.941912,-70.779573,,0,449,
```

Comment:
See comments above. PMUO 6 STC sensor ID should be 00005, PMUO 10 WFP depths should be 423

Reply:
Fixed per A.Smith comments.

Reply:
changes OK